recipes/pivy/bld.bat

```diff
@@ -0,0 +1,4 @@
+SET GENERATOR=Ninja
+
+%PYTHON% setup.py clean
```

Comment:
Do you need the `clean` step? If so this can be done using a multi-line script line and you can drop the build.sh and bld.bat files.